extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -100,6 +74,7 @@ class AmpLightbox extends AMP.BaseElement {
         this.element.style.opacity = '';
       });
     }).then(() => {
+      this.build();
```

Comment:
I would call this earlier in the function. Possibly right after the is active check. Although nothing uses `this.container_` in previous lines, it would be more defensive to have it initialized early on.